esp/scm/ws_dfu.ecm

```diff
@@ -812,7 +821,43 @@ ESPresponse [exceptions_inline, nil_remove] EraseHistoryResponse
     ESParray<ESPStruct History, Origin> History;
 };
 
+ESPStruct DFUPartLocations
+{
+    int LocationIndex;
+    string Location;
+};
+
+ESPStruct DFUPartCopies
+{
+    int PartId;
```

Comment:
@rpastrana - as above, these will always be same as position in the array that they're in.